test/integration/base.py

```diff
@@ -266,6 +266,12 @@ def bigquery_profile(self):
             }
         }
 
+    def bigquery_alternate_profile(self):
+        bigquery_profile = self.bigquery_profile()
+        execution_project = self.alternative_database
+        bigquery_profile['test']['outputs']['default2']['execution_project'] = execution_project
+        return bigquery_profile
```

Comment:
Rather than defining an additional profile, I think you can just add a new target above that also defines `execution_project`. Then you can just pass `--target alternate` as an additional argument in the test.
```python
        return {
            'config': {
                'send_anonymous_usage_stats': False
            },
            'test': {
                'outputs': {
                    'default2': {
                        'type': 'bigquery',
                        'method': 'service-account-json',
                        'threads': 1,
                        'project': project_id,
                        'keyfile_json': credentials,
                        'schema': self.unique_schema(),
                    },
                    'alternate': {
                        'type': 'bigquery',
                        'method': 'service-account-json',
                        'threads': 1,
                        'project': project_id,
                        'keyfile_json': credentials,
                        'schema': self.unique_schema(),
                        'execution_project': self.alternative_database,
                    },
                },
                'target': 'default2'
            }
        }
```

Reply:
Ok I updated that part 👍